filters/include/pcl/filters/uniform_sampling.h

```diff
@@ -72,7 +72,8 @@ namespace pcl
       typedef boost::shared_ptr<const UniformSampling<PointT> > ConstPtr;
 
       /** \brief Empty constructor. */
-      UniformSampling () :
+      UniformSampling (bool extract_removed_indices = false) :
```

Comment:
On one hand, I like this idea. On the other hand, it pushes the next release (whichever number it will have) quite far in the future. What about releasing the current state as 1.8.1 (or 1.9.0) ASAP, and then moving on with merging the current fix and other "1.9.0 milestone" features?